sdks/java/maven-archetypes/update-examples-archetype.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash -ex
+
+HERE="$(dirname $0)"
+
+EXAMPLES_ROOT="${HERE}/../../../examples/java"
```

Comment:
It is sensitive to where the script and modules are, not the directory of the user who invokes it. Seems like adding args would just make it more error prone.